src/Dolo.jl

```diff
@@ -35,17 +30,26 @@ export yaml_import, eval_with, evaluate, evaluate!, model_type, name, filename,
 
 export time_iteration, value_iteration, steady_state_residuals, simulation
 
-# set up core types
-abstract AbstractModel{ID}
-abstract AbstractSymbolicModel{ID} <: AbstractModel{ID}
-abstract AbstractNumericModel{ID} <: AbstractModel{ID}
+export ModelCalibration, FlatCalibration, GroupedCalibration
 
-typealias ASM AbstractSymbolicModel
-typealias ANM AbstractNumericModel
+# set up core types
+abstract AbstractSymbolicModel{ID}
+# abstract AbstractSymbolicModel{ID} <: AbstractModel{ID}
+abstract AbstractModel{ID} <: AbstractSymbolicModel{ID}
```

Comment:
Ahh ok I see now, thanks. That makes more sense. 

So am I correct in saying we have 

- a new symbolic model type (`SModel`) that is just a wrapper around the Dict we get when we have YAML.jl parse the model file
- A `Model` type that includes the same underlying data (actually I had a question, is the `data` field on `Model` supposed to be a Dict or a `SModel`?) and numerical objects (some just plain Julia datatypes for now) for things like the grids, domain, etc.